.gitignore

```diff
@@ -8,6 +8,7 @@
 # Ignore all logfiles, tempfiles, public assets,
 /log/*.log
 /tmp
+/vendor
```

Comment:
vendor folder is no longer used to hold third party libraries for front, therefore there is no need to add an ignore line.

Reply:
ok. will remove this and just delete the old directory from my machine